app/views/partials/_post.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= link_to user_post_path(post.author_id, post.id) do %>
+
+    <div class="post-card">
+      <h3><%= post.title %></h3>
+      <p><%= post.text[0, 200] %>...</p>
+      <div class="post-counters" >
+        <span>Likes: <%= post.likes_counter %></span>,
+        <span>Comments: <%= post.comments_counter %></span>
+      </div>
+    </div>
+    
+<% end %>
```

Comment:
- [ ] Please fix your posts and comments when I create a new post it gives this error:`expected, got 1 which is an instance of Integer` so kindly fix it :slightly_smiling_face: 

![Screenshot from 2023-10-05 20-32-02](https://github.com/Dorcas126/blog-app-rails/assets/45226848/cb7f236a-9cbe-46e9-925e-37fdca575729)